lib/template-mixins.js

```diff
@@ -7,19 +7,35 @@ var Hogan = require('hogan.js'),
     _ = require('underscore'),
     moment = require('moment');
 
+var deprecate = require('deprecate');
```

Comment:
https://github.com/dougwilson/nodejs-depd looks like a more robust deprecation library, that mirrors the behaviour of debug (which we're already using for some of our logging messages).

In particular:

> This message will appear once per unique caller site. Caller site is the first call site in the stack in a different file from the caller of this function.

This would allow us to more easily identify usages of this deprecated functionality..